controllers/user_controller.go

```diff
@@ -0,0 +1,59 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type UserController interface {
+	Get(context *gin.Context)
+}
+
+type userController struct {
+}
+
+func NewUserController() UserController {
+	return &userController{}
+}
+
+func (ctrl *userController) Get(context *gin.Context) {
+	// Id refers to MongoDB user object ID. Specificty refers to the verbocity of the returned friends array
+	// 0 returns ONLY friend user IDs (default), 1 returns friend user IDs AND usernames
+	type IGet struct {
+		ID          string `form:"id" binding:"required"`
```

Comment:
Could potentially remove the binding from this and default to the authenticated user (since this route is auth-protected by the middleware). Just food for thought, if we want to do this it can def go in different issue though

Reply:
I think this is a good idea, it would be really convenient to just default to the authenticated user. I'll create a ticket for it and we can def work on it in a different PR.